frontend/src/pages/MapViewPage.js

```diff
@@ -54,37 +54,46 @@ class MapViewPage extends Component {
       <div>
         <Script url = {this.url} onLoad = {this.handleScriptLoad}/>
         <TopNavbar history={this.props.history} loggedIn={this.state.loggedIn} plus_code={this.state.plus_code}/>
-        <Toast style={{position: "absolute", zIndex: 2, padding: "0rem", minWidth: "40rem", float: "right", margin: "1rem"}}>
-        <Toast.Body>	
-          <Form style={{marginLeft: '3rem', marginRight: '3rem'}}>
-          <Col xs="auto">
-            <Form.Group>
-              <Form.Control id = "autocomplete" placeholder = "Enter university"/>
-            </Form.Group>
-            <Form.Group>
-              <Form.Control
-                onChange={this.handleFilter}
-                as="select"
-                className="my-1 mr-sm-2"
-                id="categoriesSelect"
-                custom="true">
-                <option value="">Filter</option>
-                <option value="0">Social Gathering</option>
-                <option value="1">Volunteer Event</option>
-                <option value="2">Student Organization Event</option>
-              </Form.Control>
-            </Form.Group>
-            <Form.Group>
-              <Form.Check
-                onChange={this.handleCheckboxChange}
-                defaultChecked={this.isChecked}
-                type="checkbox"
-                label="Show today's events"/>
-            </Form.Group>
-          </Col>
-          </Form>
-        </Toast.Body>
-        </Toast>	
+	  <Toast style={{backgroundColor: "white", position: "absolute", zIndex: 2, border: 0, borderRadius: "1rem", padding: 0, minWidth: "25rem", float: "right", margin: "1rem"}}>
+	    <Toast.Body>
+      	      <Card style={{border: 0}}>
+		<Form>
+		  <Form.Control style={{border: 0, focusOutline: "none"}} id = "autocomplete" placeholder = "Enter university"/>
+		</Form>
+      	      </Card>
+              <hr/>
```

Comment:
Incorrect indentation

Reply:
Github shows different indentation on the "files changed" tab than the actual file. If you view the file itself, the indentation is fine.